skimage/feature/tests/test_corner.py

```diff
@@ -128,8 +130,7 @@ def test_hessian_matrix_eigvals_3d(im3d):
 
     E0, E1, E2 = E[:, E.shape[1] // 2]  # cross section
     row_center, col_center = np.array(E0.shape) // 2
-    circles = [draw.circle_perimeter(row_center, col_center, radius,
-                                     shape=E0.shape)
+    circles = [circle_perimeter(row_center, col_center, radius, shape=E0.shape)
```

Comment:
And a small suggestion: you changed the imports in `test_corner.py`, e.g. `skimage import draw` to `from skimage.draw import ...`. Was there a reason behind that? Otherwise it would probably be best to keep the "diff" as small as possible and refrain from stylistic changes that don't touch the code you're modifying anyway. I'm myself a huge offender in that regard so perhaps that's why I noticed it. :sweat_smile: 

The whitespace fixes are a clear improvement though.

Reply:
I first wanted to make the imports respect the suggestion of [the python style guide](https://www.python.org/dev/peps/pep-0008/#imports), but I confess that I was tempted by the stylistic changes that you catched :smile:.
I can revert it if you wish.

Reply:
I first wanted to make the imports respect the suggestion of the python style guide, but I confess that I was tempted by the stylistic changes that you catched smile.
I can revert it if you wish.

Reply:
I agree with @lagru. Best to keep diffs small. In some cases, it is worth making the changes, but then they should be done in their own PR. Later when one is doing `git bisect` to find when a thorny bug appeared, small diffs are extremely appreciated!

Reply:
I reverted the stylistic changes ;-)